examples/adding_function.cz

```diff
@@ -0,0 +1,13 @@
+~Prompt "First number: " to and set input a~
+int a = sys.read("First number: ")
+~Prompt "Second number:" to and set input b~
+int b = sys.read("Second number: ")
+
+~Define add function~
+fun add(fnum, snum){
```

Comment:
Also, declare types of variables with `fun add(int fnum, int snum) -> int`

Reply:
Got it